apps/asap-server/src/entities/user.ts

```diff
@@ -59,37 +59,41 @@ export const userUpdateSchema = Joi.object({
   .min(1)
   .required();
 
-type GraphqlUserTeam = NonNullable<
+export type GraphqlUserTeam = NonNullable<
   NonNullable<FetchUserQuery['findUsersContent']>['flatData']['teams']
 >[number];
-export const parseGraphQLUserTeamConnection = (
-  item: GraphqlUserTeam,
-): UserTeam => {
-  if (item.id === null) {
-    throw new Error('Invalid user-team connection');
-  }
 
-  const team = item.id[0];
-  if (!team) {
-    throw new Error(`User team connection is not defined`);
-  }
-  const displayName = team.flatData?.displayName;
-  const proposal = team.flatData?.proposal;
-
-  if (!item.role || !isTeamRole(item.role)) {
-    throw new Error(`Invalid team role: ${item.role}`);
-  }
+export const parseGraphQLUserTeamConnections = (
```

Comment:
im not a huge fan of exposing functions only for tests - this could just be internal to the client and tested in the client test. We are about to drop the nock fixtures anyway for all entities which will make these tests much leaner.
But for now leave it as is